tutorials/text/annotations.py

```diff
@@ -104,6 +113,24 @@
 # Advanced Annotations
 # --------------------
 #
+# Placing Annotations Relative to Data
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Annotations can be positioned at a relative offset to the *xy* input to
+# annotation by setting the *textcoords* kwarg to one of the offset options.
+# In this example, the annotations are offset 1.5 physical points from the
```

Comment:
```suggestion
# In this example, the annotations are offset 1.5 physical points (=1.5 / 72 inches) from the
```

Reply:
I would just write this out most places. We could also clarify in the transforms tutorial that "points" means "typographic points". I think the confusion arises if folks don't know that fontsize is in "points" and "points" sounds a lot like pixels. We could also just link out to wikipedia https://en.wikipedia.org/wiki/Point_(typography)